nuke-from-orbit/loadtester

```diff
@@ -299,6 +308,27 @@ set_iap_bindings() {
   done
 }
 
+compare_tags() {
+  current_tag=$(kubectl get deployment lw-pod -o json | jq -r '.spec.template.spec.containers[0].image' | cut -f 2 -d ':')
+  echo "Updating tag from ${current_tag} to ${image_tag}"
+  if [[ $current_tag == $image_tag ]]
+  then
+    printf "\n${RED}Error: Provided tag matches current tag. Please enter a unique tag.${NC}\n"
```

Comment:
Consider allowing reuse of the same tag name to simplify the developer experience

Reply:
i like being explicit about updating as it works better with kubernetes' image pull policies, but a good option may be to allow reuse of the 'latest' tag... that's what it's for after all...